aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -571,23 +604,43 @@ Resources:
 <%   end -%>
 <% end-%>
 
+
 <% if alarms-%>
+  # ---------------------------------------------
+  # CloudWatch Alarms
+  # ---------------------------------------------
 <%=  component 'alarms'%>
 <% end -%>
 
+
+
 <% if database-%>
+  # ---------------------------------------------
+  # Database
+  # ---------------------------------------------
 <%=  component 'database'%>
 <% end -%>
 
+  # ---------------------------------------------
+  # Codeprojects Resources
+  # ---------------------------------------------
 <%= resource_component 'codeprojects_resources' %>
 
+  # ---------------------------------------------
+  # Custom Chef Configuration Secret
+  #
+  # Update this secret string manually after the
+  # stack is created to customize Chef
+  # configuration. The value should never be
+  # changed in the stack itself, since it would
+  # clobber all other changes.
```

Comment:
super tiny nit: this probably doesn't need to be wrapped so aggressively. Unless it's important that all this text remain strictly within the `---` headings, in which case the TODOs above should be wrapped.